chime/repo_functions.py

```diff
@@ -531,6 +535,39 @@ def save_working_file(clone, path, message, base_sha, default_branch_name):
 
     return clone.active_branch.commit
 
+def get_conflict(clone, other_branch_name):
+    ''' Attempt to merge from origin default branch, return a conflict (if any).
+    
+        Currently requires a clean working tree in the clone!
+    '''
+    clone.git.fetch('origin', other_branch_name)
+
+    try:
+        # Try a no-commit merge from the other branch.
+        clone.git.merge(_origin(other_branch_name), '--no-commit')
+
+    except GitCommandError:
```

Comment:
Try to log (info) some information about the error, in case it's not a merge conflict.